src/components/Auth/SignUpForm.schema.js

```diff
@@ -0,0 +1,33 @@
+import Joi from '@hapi/joi';
+import { createValidationResolver } from '../form';
+
+const schema = Joi.object({
+  firstName: Joi.string()
+    .allow('')
+    .trim(),
+  lastName: Joi.string()
+    .allow('')
+    .trim(),
+  username: Joi.string()
+    .alphanum()
+    .min(3)
+    .max(30)
+    .required()
+    .trim(),
+  email: Joi.string()
+    .email({ tlds: { allow: false } })
+    .required()
+    .trim(),
+  password: Joi.string().required(),
+});
+
+const defaultValue = {
```

Comment:
suggest to rename to `defaultValues`

Reply:
Done: https://github.com/codebuddies/frontend/pull/105/commits/027abd780c439e1de707ee31383b20040cceb592